src/elements/TableCell/TableCell.js

```diff
@@ -123,10 +124,17 @@ const TableCell = ({
     )
   } else if (data.type === 'path') {
     return (
-      <div className={`table-body__cell ${data.class}`}>
+      <div
+        className={`table-body__cell ${data.class}`}
+        onClick={() => copyToClipboard(data.value.path)}
+      >
         <Tooltip
           className="data-ellipsis text_small"
-          template={<TextTooltipTemplate text={data.value.path} />}
+          template={
+            <TextTooltipTemplate
+              text={`Click to copy - ${data.value.path}`}
+            />
```

Comment:
Prettier has an error here.
It wants this to be:
```suggestion
            <TextTooltipTemplate text={`Click to copy - ${data.value.path}`} />
```